### PR TITLE
Don't run ApiCompat and InheritDoc in test builds

### PR DESCRIPTION
**Pull request by pakrym**
Save some build time.

### CODE REVIEW

**Review by christothes — APPROVED** [review]
Just curious - how much time did Inheritdoc save?

**Comment by pakrym** [comment]
>Just curious - how much time did Inheritdoc save?

That we would see 😄 But it takes almost 15minutes just to build tests. Pretty sure there is something we can save there.

**Comment by pakrym** [comment]
`ProjectRef_ubuntu1804_net50` went from 30 to 24 minutes. I call it good enough.

**Comment by christothes** [comment]
> `ProjectRef_ubuntu1804_net50` went from 30 to 24 minutes. I call it good enough.

Those minutes add up - I'll take it 😄

File: eng/pipelines/templates/jobs/ci.tests.yml [hunk 1]
```diff
@@ -53,6 +53,7 @@ jobs:
           /p:SDKType=${{ parameters.SDKType }}
           /p:ServiceDirectory=${{ parameters.ServiceToTest }}
           /p:IncludeSrc=false /p:IncludeSamples=false /p:IncludePerf=false /p:IncludeStress=false
+          /p:RunApiCompat=false /p:InheritDocEnabled=false
```
Comment (weshaggard) [inline eng/pipelines/templates/jobs/ci.tests.yml:56]: I assume these are already disabled for test projects but this disables them for all the library projects just for the test run in CI for the.

**Pull request merged**